src/entities/data-address.ts

```diff
@@ -1,5 +1,5 @@
 export interface BaseDataAddress {
-  [key: string]: string | undefined;
+  [key: string]: string | undefined | any;
```

Comment:
Why would we have `any`?

Reply:
IDK, I think @OlfaBensoussia did this, anyway without this there's an hardly understandable stack trace in the test results :smiling_face_with_tear:  will solve it later as well, the type structure of Asset/DataAddress is way too complicated currently

Reply:
@fdionisi I made this change when I was working on updating the `AssetInput` [here](https://github.com/Think-iT-Labs/edc-connector-client/blob/a684f77b5e2eb5d80ac6fc305ce21f4fc910ea91/src/entities/asset.ts#L20C1-L20C1). What pushed me to do it is that the connector was supported data address definition in those two different ways:
1- 
```
{
    "@context": {},
    "asset": {
        "@type": "Asset",
        "@id": "{{ASSET_ID}}"
    },
    "dataAddress": {
        "@type": "DataAddress",
         "properties": {
            "type": "HttpData",
            "description": "Product EDC Demo Asset",
            "baseUrl": "https://jsonplaceholder.typicode.com/todos"
        }
    }
}
```
2- 

```
{
    "@context": {},
    "asset": {
        "@type": "Asset",
        "@id": "{{ASSET_ID}}2"
    },
    "dataAddress": {
        "@type": "DataAddress",
        "type": "HttpData",
        "description": "Product EDC Demo Asset",
         "properties": {
            "baseUrl": "https://jsonplaceholder.typicode.com/todos"
        }
    }
}
```

Reply:
this will change in 0.2.0, or better, there is a `/v3` asset endpoint with a clearer api:
https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.2.0#/Asset/createAsset_1